src/float32sr.jl

```diff
@@ -141,4 +143,20 @@ function Base.bitstring(x::Float32sr,mode::Symbol)
     else
         return bitstring(x)
     end
+end
+
+# BIGFLOAT
+Base.BigFloat(x::Float32sr) = Float32_stochastic_round(Float64(x))
+Float32sr(x::BigFloat) = Float32sr(Float64(x))
+
+
+function Base.decompose(x::Float32sr)::NTuple{3,Int}
```

Comment:
Why not just `Base.decompose(x::Float32sr) = Base.decompose(Float32(x))`?

Reply:
Can you clarify what you mean @milankl ? You mean in place of the function on L152?

Reply:
Yes, you've been redefining the `decompose` function, although Float32/Float32sr have an identical bitwise encoding. So I'd just call the existing function via reinterpretation from Float32sr to Float32.

Reply:
👍